client/my-sites/stats/site.jsx

```diff
@@ -192,6 +203,7 @@ class StatsSite extends Component {
 								'Accept payments for just about anything and turn your website into a reliable source of income with payments and ads.'
 							) }
 							href={ `/earn/${ slug }` }
+							dismissPreferenceName={ hasInstantWordAds ? `stats-earn-nudge-${ siteId }` : null }
```

Comment:
Is there a reason we are using `siteId` here? I am not sure we needed as per docs
https://github.com/Automattic/wp-calypso/blob/a87658f46866d2455eb5d8f8c02df4da284cf014/client/blocks/dismissible-card/README.md#L15

Also, can you please rename `stats-earn-nudge` to `stats-earn-nudge-wordads` since this banner appears only for users not having wordads
https://github.com/Automattic/wp-calypso/blob/21f1732978ee60fb363fdafaff62db224a6c4b96/client/my-sites/stats/site.jsx#L197

And last but not least, rename `hasInstantWordAds` to `planSupportsWordAdsInstantFeature`

Reply:
> Is there a reason we are using siteId here?

It's basically to make the nudge dismiss per site rather than at an account level. Let me know if that should be changed though. :)

Reply:
You are totally right! I thought that it would be automatically handled but this is not the case... I just made a test right now and as you said, we actually save the preference in the account level `/rest/v1.1/me/preferences` so appending the `siteId` in the key is needed in order to keep this setting per site.